index.js

```diff
@@ -111,28 +114,89 @@ Please ensure the parent directory is writable, or set a different path with -d`
             warn('Device setup requires parameter --otc to be 8 or more characters')
             quit(null, 2)
         }
-        info('Entering Device setup...')
+        print('Starting Device setup...')
         if (!options.ffUrl) {
             warn('Device setup requires parameter --ff-url to be set')
             quit(null, 2)
         }
-        AgentManager.quickConnectDevice().then((success) => {
-            if (success) {
-                const runCommandInfo = ['flowfuse-device-agent']
-                if (options.dir !== '/opt/flowfuse-device') {
-                    runCommandInfo.push(`-d ${options.dir}`)
+        let deviceSettings = null
+        AgentManager.quickConnectDevice().then((provisioningData) => {
+            deviceSettings = provisioningData
+            if (!deviceSettings) {
+                warn('Device setup was unsuccessful')
+                quit(null, 2)
+            }
+            const runCommandInfo = ['flowfuse-device-agent']
+            if (options.dir !== '/opt/flowfuse-device') {
+                runCommandInfo.push(`-d ${options.dir}`)
+            }
+            print('Success! This Device can be launched at any time using the following command:', figures.tick)
+            print(runCommandInfo.join(' '), ' ')
+            if (!options.otcNoImport) {
+                // Support for importing flows during initial state check-in was added after 2.15.0.
+                const ffVersion = deviceSettings.meta?.ffVersion?.replace(/[^0-9.]/g, '') || '0.0.0' // Strip suffixes like -beta.1
+                const ffSupportsImport = (ffVersion && semver.gt(ffVersion, '2.15.0'))
```

Comment:
```suggestion
                // Support for importing flows during initial state check-in was added after 2.16.0.
                const ffVersion = deviceSettings.meta?.ffVersion?.replace(/[^0-9.]/g, '') || '0.0.0' // Strip suffixes like -beta.1
                const ffSupportsImport = (ffVersion && semver.gt(ffVersion, '2.16.0'))
```

Current version of FF is 2.16 - this should check for > 2.16